<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkAddDinoWidget</class>
 <widget class="QWidget" name="BulkAddDinoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk add dinos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appstyle.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="60,40" columnminimumwidth="0,0">
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="40,60">
          <item row="1" column="1">
           <widget class="QSpinBox" name="probabilitySpin">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="probabilityLabel">
            <property name="text">
             <string>Probability:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="settingsTooltip">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Dinos template</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9" columnstretch="60,40">
        <item row="0" column="0">
         <widget class="QLabel" name="templateTooltip">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_8" columnstretch="40,0">
          <item row="2" column="0">
           <widget class="QLabel" name="isBabyLabel">
            <property name="text">
             <string>Is baby:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="isNeuteredLabel">
            <property name="text">
             <string>Is neutered:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="imprintAmountSpin">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="imprintAmountLabel">
            <property name="text">
             <string>Imprint amount:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="isNeuteredComboBox">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>True</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>False</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="levelLabel">
            <property name="text">
             <string>Level:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="isBabyComboBox">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>True</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>False</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="levelSpin">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Blueprints</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="blueprintsEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="50,25,25">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submitButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="appstyle.qrc"/>
 </resources>
 <connections/>
</ui>
